website/utils/test/url_utils.test.ts

```diff
@@ -529,6 +535,31 @@ describe("Viewer state serialization", () => {
   });
 });
 
+describe("Camera state", () => {
+  it("uses default camera state when choosing elements to exclude/ignore", () => {
+    let cameraState: CameraState = {
+      ...getDefaultCameraState(),
+    };
+    // No changes from default
+    expect(serializeCameraState(cameraState, true)).toEqual(undefined);
+
+    cameraState = { ...cameraState, position: [1, 2, 3] };
+    expect(serializeCameraState(cameraState, true)).toEqual("pos:1:2:3");
+  });
+
+  it("default camera state has not been changed", () => {
+    // The default camera state should NOT change unless backwards compatibility
+    // is added to ensure old links still work.
```

Comment:
the comment doesn't really say why this would break old links if it changed.  is it just because they would show up at a different view orientation?

Reply:
Yeah, they'll be oriented differently. Whether this is a breaking issue depends on the use case, but I can imagine someone doing an exercise where they want the exact current default view for like, snapshot comparison or something.